content/post/gld.md

```diff
@@ -0,0 +1,1355 @@
+to query GitHub API. However, we were not satisfied with
+it's detection quality: many projects which actually contain the license file in a non-standard
+format are missed, and some are misclassified. This is how
+[go-license-detector](https://github.com/src-d/go-license-detector) was born.
+
+The goals were defined from the very beginning:
+
+1. Favor detection rate to the classification accuracy (target data mining instead of compliance).
+2. Perform fast.
+3. Detect as many licenses as possible on the [hand-collected dataset of 1,000 top-starred repositories
+on GitHub](https://github.com/src-d/go-license-detector/blob/master/licensedb/dataset.zip).
+4. Comply with SPDX [licenses list](https://github.com/spdx/license-list-data) and
+[detection guidelines](https://spdx.org/spdx-license-list/matching-guidelines).
+
+(1) means that we should rather label a project with a bit inaccurate license than miss it's
+license completely. The open source compliance departments will not be satisfied with this choice,
+as they need the opposite: the missed projects are manually studied. (2) restricts from using a
+scripting language such as Python or Ruby, and we chose Go for our implementation. (3) paves
+the way to many technical tricks, hacks and heuristics and enables any hardcore and complex code.
+(4) is the only way to obtain the database of 400 different licenses validated by professional
+lawyers.
+
+The following table compares the current go-license-detector with GitHub's built-in license detector,
+Google's licenseclassifier and Ben Boyter's `lc` on the
+[reference 1k dataset](https://github.com/src-d/go-license-detector/blob/master/licensedb/dataset.zip):
+
+|Detector|Detection rate|Time to scan, sec|
+|:-------|:----------------------------------------:|:-----------------------------------------|
+|[go-license-detector](https://github.com/src-d/go-license-detector)| 99% \\(\\quad(\\frac{897}{902})\\) | 16 |
+|[benbalter/licensee](https://github.com/benbalter/licensee)| 75% \\(\\quad(\\frac{673}{902})\\) | 111 |
+|[google/licenseclassifier](https://github.com/google/licenseclassifier)| 76% \\(\\quad(\\frac{682}{902})\\) | 907 |
+|[boyter/lc](https://github.com/boyter/lc)| 88% \\(\\quad(\\frac{797}{902})\\) | 548 |
+
+The total number of repositories in the dataset is 958, however, only 902 contain any pointer to
+the license - we looked though each of them. The rest are mainly "awesome lists" and Chinese projects
+and translations of the western books. We filed issues for the maintainers to clarify the license
+[\[1\]](https://github.com/DrkSephy/es6-cheatsheet/issues/90)
+[\[2\]](https://github.com/kdn251/interviews/issues/63)
+[\[3\]](https://github.com/markerikson/react-redux-links/issues/87)
+[\[4\]](https://github.com/CodeHubApp/CodeHub/issues/441)
+[\[5\]](https://github.com/h4cc/awesome-elixir/issues/4400)
+[\[6\]](https://github.com/fffaraz/awesome-cpp/issues/449).
+Few added the license; some maintainers even demonstrated aggression.
```

Comment:
No need to mention this, it's a negative aspect that doesn't add much to the post.